package.json

```diff
@@ -141,6 +141,7 @@
     "react-icons": "^4.12.0",
     "react-intersection-observer": "^9.5.2",
     "react-loading-skeleton": "^3.3.1",
+    "react-markdown": "^9.0.1",
```

Comment:
We're still using remoteMdx (with compileMDX) in other parts of the website. I'm not against replacing it but we should probably just stick to one? What pushed you to change?

Reply:
Good point - I first thought I had to replace some tags by components, but this wasn't the case. 
-> removed react-markdown